src/commands/create.js

```diff
@@ -1,26 +1,39 @@
 const { ApifyCommand } = require('../lib/apify_command');
 const { flags: flagsHelper } = require('@oclif/command');
-const { DNS_SAFE_NAME_REGEX } = require('apify-shared/regexs');
 const fs = require('fs');
 const path = require('path');
 const copy = require('recursive-copy');
 const inquirer = require('inquirer');
 const execWithLog = require('../lib/exec');
 const outputs = require('../lib/outputs');
 const { updateLocalJson } = require('../lib/files');
-const { setLocalConfig, setLocalEnv, getNpmCmd } = require('../lib/utils');
+const { setLocalConfig, setLocalEnv, getNpmCmd, validateActorName} = require('../lib/utils');
 const { ACTS_TEMPLATES, DEFAULT_ACT_TEMPLATE, EMPTY_LOCAL_CONFIG, ACTS_TEMPLATE_LIST } = require('../lib/consts');
 
 class CreateCommand extends ApifyCommand {
     async run() {
         const { flags, args } = this.parse(CreateCommand);
-        const { actorName } = args;
+        let { actorName } = args;
         let { template } = flags;
 
         // Check proper format of actorName
-        if (!DNS_SAFE_NAME_REGEX.test(actorName)) {
-            throw new Error('Name of your actor, ' +
-                'must be a DNS hostname-friendly string(e.g. my-newest-actor).');
+        if (!actorName) {
+            const actorNamePrompt = await inquirer.prompt([{
+                name: 'actorName',
+                message: 'Actor name:',
```

Comment:
Maybe `Name of the new actor (and directory):` to make it clear it will be created?